src/autocomplete/databases/redis/grammar/RedisLexer.g4

```diff
@@ -36,7 +53,8 @@ DOUBLE_QUOTE : '"';
 
 fragment DECIMAL_DIGIT: [0-9];
 
-DECIMAL_LITERAL: DECIMAL_DIGIT+;
+POSITIVE_DECIMAL_LITERAL : DECIMAL_DIGIT+;
+DECIMAL_LITERAL          : [-]? POSITIVE_DECIMAL_LITERAL;
```

Comment:
What does it mean? is `-4-5-6` being parsed as a `POSITIVE_DECIMAL_LITERAL`?

Reply:
`POSITIVE_DECIMAL_LITERAL` is just positive numbers, like `456`
`DECIMAL_LITERAL` is both positive and negative
`-4-5-6` won't work either way, because our minus is only at the start